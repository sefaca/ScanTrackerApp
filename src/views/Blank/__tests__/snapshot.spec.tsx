// THIS IS A GENERATED FILE. DO NOT MODIFY
import {snapshotTest} from '@storybook-utils';
const imports = require('../index.stories') as Record<string, any>;

Object.entries(imports)
  .filter(([key]) => key !== 'default')
  .forEach(([key, value]) => {
    snapshotTest(key, value);
  });
