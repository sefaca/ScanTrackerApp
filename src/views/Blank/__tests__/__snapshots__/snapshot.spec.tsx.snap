// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ScreenMainBase story 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      [
        {},
      ]
    }
  >
    Hi!
  </Text>
</View>
`;
